packages/app/template/index.js

```diff
@@ -151,7 +151,10 @@ async function createApp (ssrContext) {
   }
 
   <% if (store) { %>
-  if (process.client) {
+  if (process.client || process.browser) {
+    if (process.browser) {
```

Comment:
Would you please add a `<% if (isDev) { %> ... <% } %>` condition around all `if (browser)` checks? Keeping it, adds a little extra overhead to production builds.